modules/recast/navigation_mesh_generator.cpp

```diff
@@ -435,8 +449,14 @@ void EditorNavigationMeshGenerator::bake(Ref<NavigationMesh> p_nav_mesh, Node *p
 
 	ERR_FAIL_COND(!p_nav_mesh.is_valid());
 
-	EditorProgress ep("bake", TTR("Navigation Mesh Generator Setup:"), 11);
-	ep.step(TTR("Parsing Geometry..."), 0);
+	EditorProgress *ep = NULL;
+
+	bool is_in_editor = Engine::get_singleton()->is_editor_hint();
```

Comment:
It's probably not worth creating a variable if you're going to use it just once below.